### PR TITLE
Add footer styles

### PR DESCRIPTION
**Pull request by acacheung**
Fix location top spacing
#100

### CODE REVIEW

**Comment by acacheung** [comment]
@Ctannerweb :eyes:

File: 2016/styles/modules/footer.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
+.footer {
+  background-color: var(--black);
+  color: var(--white);
+  a {
+    color: var(--gray);
+    transition-duration: .25s;
+    transition-property: opacity;
```
Comment (Ctannerweb) [inline 2016/styles/modules/footer.css]: Is the transition requested by design? Thinking back to design sprints we decided that transitions on `a` tags was not needed?

**Comment by Ctannerweb** [comment]
Are we missing social links?

File: colorado/index.html [hunk 2]
```diff
@@ -57,32 +57,32 @@
           <h2 class="footer__heading">UX Camp 2016</h2>
           <ul class="footer__contacts">
             <li class="footer__contact">
-              <span>Email:</span>
-              <a href="mailto:UXCamp@gmail.com">UXCamp@gmail.com</a>
+              <span class="footer__contact__key">Email:</span>
+              <a class="footer__contact__value spot-color" href="mailto:UXCamp@gmail.com">UXCamp@gmail.com</a>
```
Comment (Ctannerweb) [inline colorado/index.html]: Colorado spot-color in the footer is not very readable, should we waiting to change this depending on more design changes?

File: 2016/styles/modules/footer.css (hunk 1 above)
Comment (acacheung) [inline 2016/styles/modules/footer.css]: will review this with tim during qa

File: colorado/index.html (hunk 2 above)
Comment (acacheung) [inline colorado/index.html]: the colors haven't been finalized yet. will be updated :)

**Pull request merged**